### ISSUE TITLE
Activity stays on default "TV Kijken"

### ISSUE THREAD

**Issue by RolfvanderKwaak** [rkokkelk/com.jreenen.homeyharmonyhub#92]
Hi,

First compliments to this wonderful homey app, many thank!!

Now my issue;
When I compare an Activity with hub/activity it is always not true only works "yes" on "TV Kijken".

Where?
I set up a flow
1. Select Harmony Hub - Activiteit gestart (of Activiteit wordt gestart / ook geprobeerd)
2. Activiteit is gelijk aan (activiteit) , hub, PowerOff
3. the actual action (eg. a logical operator)

When I select the tag "Activiteit" in step 2 it is only suggesting (or showing?) TV Kijken.
In step 2 I can select the actual Activities as defined in the Harmony Hub.
With testing it shows "Tv Kijken"

What I expect that the tag "Activiteit" will be populated with the real value of the current activity present in Harmony Hub.

What is working:

The activity buttons (PowerOff, Radio, Ziggo) are having the right status. PowerOff is "On"  when the power is off. All other activities are 'Off" / greyed out. When I click "Radio" it really works and the button is flipped on and PowerOff goes off.

Last version of Homey Pro 4.2.0 and your app 2.0.7

Any help here possible? Or am I doing something wrong?
Thanks,
Rolf
![Screenshot_20201126-150405_Homey](https://user-images.githubusercontent.com/75081287/100360933-dad9c880-2ff9-11eb-9577-d17956242739.jpg)
![Screenshot_20201126-150341_Homey](https://user-images.githubusercontent.com/75081287/100360937-dc0af580-2ff9-11eb-9c23-46bd65c163e0.jpg)
![Screenshot_20201126-150104_Homey](https://user-images.githubusercontent.com/75081287/100360944-dca38c00-2ff9-11eb-844e-a77e06a165f2.jpg)
![Screenshot_20201126-150101_Homey](https://user-images.githubusercontent.com/75081287/100360948-dd3c2280-2ff9-11eb-970f-567b7da85f9c.jpg)
![Screenshot_20201126-150053_Homey](https://user-images.githubusercontent.com/75081287/100360951-ddd4b900-2ff9-11eb-947a-7435159bb7d3.jpg)

Screenshots attached

**Comment by RolfvanderKwaak** [comment]
The thing here is; when you change your activity - the changing is triggering Homey. So the homey app notices a starting activity. Only the activity itself (the content of the variable when you evaluate it) stays or reverts back to the "default one" TV Kijken.

Eg. starting Radio is seen:
![image](https://user-images.githubusercontent.com/75081287/121912863-35860800-cd31-11eb-99cc-8947d9e862be.png)

But the status, when you evaluate the Harmony Hub activity, remains TV Kijken.

I have now created Homey logical expressions to pin down which activity is started. "T start" is triggered by device denon (powerchange) and by the Harmony activity.
For now I know which activity is active so I can start from here with fine tuning that activity. At this stage I only set a textual variable but I will change this in the future to what I need to adjust;
- volume of an activity related to the time of the day or the fact that I am working from home (eg. Radio on low volume versus normal volume in the weekend or after work hours).
- Channel of Ziggo on 3 when kids start this activity in the morning but on Saturday on Disney Channel. And in e.g. in the evening start Veronica on Thursdays.
- Channel of TV Kijken on the Satelite dish on 178 on Tuesdays and Thursdays and 411 on Wednesdays after 20:00

![image](https://user-images.githubusercontent.com/75081287/121913526-c3fa8980-cd31-11eb-9bb3-62f48d60a49c.png)

Example of one flow:
![image](https://user-images.githubusercontent.com/75081287/121913315-94e41800-cd31-11eb-8e40-8310852b2852.png)
